opendebates/tasks.py

```diff
@@ -111,8 +111,8 @@ def update_trending_scores():
         with connection.cursor() as cursor:
             cursor.execute(sql)
             cursor.execute(sql2)
-    except:
-        logger.exception("Unexpected error in update_trending_scores")
+    except Exception as error:
+        logger.exception("Unexpected error in update_trending_scores: {}".format(error))
```

Comment:
No need for this change. logger.exception always includes the information from the current exception.

Reply:
The main reason for me making this change was that a bare `except:` no longer passes our `flake8` check with `E722 do not use bare 'except'`. Maybe I can just not log the error itself, since that would be redundant.

Reply:
> Maybe I can just not log the error itself, since that would be redundant.

+1